File: LeetCode Solutions/Problems/9. Palindrome Number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        temp = x
        reverseNumber = 0
        while x > 0:
            remainder = x % 10
            reverseNumber = (reverseNumber * 10) + remainder
            x = x // 10
        if temp == reverseNumber:
            return True
        else:
            return False

File: LeetCode Solutions/Problems/9. Palindrome Number.c
bool isPalindrome(int x){
    long reverseNumber = 0;
    int temp = x;
    while (x > 0){
        int remainder = x % 10;
        reverseNumber = reverseNumber * 10 + remainder;
        x /= 10;
    }
    if (temp == reverseNumber){
        return true;
    }
    else{
        return false;
    }
}